container/options.go

```diff
@@ -216,6 +218,19 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the heights of the top and the bottom containers.
+// The first container will be fixed and the second one will fill the rest of the space.
+// Allows values greater or equal to zero.
+func SplitFixed(cells int) SplitOption {
+	return splitOption(func(opts *options) error {
+		if cells < 0 {
```

Comment:
I see that SplitPercent does the same thing, but it probably is worth asking: Is it valid to specify SplitFixed(0) ? Or should it be an error.

Setting it to zero is the same thing as not performing the split at all. What do you think?

Reply:
I think we should support SplitFixed(0) and SplitPercent(0).

It doesn't seem to hurt to support 0, and it maybe can be used to hide and show the container.
I.e., instead of unmounting and mounting a container when wanting to show and hide, you can maybe just set the value to zero and then set it to bigger than zero.

So supporting zero might have some functionality.

Reply:
That is a good point, thank you for providing the example.

Let's keep supporting zero as you suggested.